lib/src/services/ds_bottom_sheet.service.dart

```diff
@@ -23,15 +22,20 @@ class DSBottomSheetService {
       child: Stack(
         children: [
           controller != null
-              ? ListView(
-                  controller: controller,
+              ? Column(
                   children: [
-                    StickyHeader(
```

Comment:
you can't remove the `StickyHeader` as this was a solution I found so that the `grabbler` can drag the bottomsheet up.

it creates a fixed item inside a list